homeassistant/components/tractive/__init__.py

```diff
@@ -284,11 +284,16 @@ def _send_activity_update(self, event: dict[str, Any]) -> None:
         )
 
     def _send_wellness_update(self, event: dict[str, Any]) -> None:
+        sleep_day = None
+        sleep_night = None
```

Comment:
It probably doesn't matter here but I'd probably use a local var for `event["sleep"]` so you don't have to do the lookup many times.

Reply:
Good idea, I'm planning a small refactoring of Tractive and will follow your suggestion.